debian/scripts/usr/local/bin/config_save

```diff
@@ -0,0 +1,36 @@
+# Save current configurations to partition containing conf.d
```

Comment:
@hamishcoleman Where shall I place this file?

Reply:
There isnt really a well named place for non-debian customisations in the repo (I had some ideas, but it didnt come up enough to fix yet) - I think you found the debian/packages.d/systemd.customise.add/usr/local/sbin/ dir which should work for now.

Reply:
It wouldn't automatically run this right? And bin or sbin in this case? System never runs it, it's the administrator of the nodes that may use it.

Reply:
sbin is for tools that the administrator runs - so it is definitely sbin

Reply:
Local is a bit of a toss up for me. By definition it sorta applies sorta does not.

>The /usr/local hierarchy is for use by the system administrator when installing software locally. It needs to be safe from being overwritten when the system software is updated. It may be used for programs and data that are shareable among a group of hosts, but not found in /usr. 

Since were not "upgrading" we don't really need local?